reana_workflow_engine_yadage/tracker.py

```diff
@@ -65,14 +65,26 @@ def finalize(self, adageobj) -> NoReturn:
         # needed to comment it here due to a hack in cli.py
         # self._publish_workflow_final_status()
 
-    def _publish_workflow_final_status(self):
+    def publish_workflow_running_status(self) -> None:
+        """Send MQ to indicate running status for tracked workflow."""
+        try:
+            log.debug("Publishing workflow running status...")
+            self.publisher.publish_workflow_status(
+                self.workflow_id, status=int(RunStatus.running)
+            )
+        except Exception as e:
+            log.error(f"Workflow running status publish failed: {e}")
+            raise e
+
+    def publish_workflow_final_status(self) -> None:
+        """Send MQ to indicate finished or failed workflow."""
```

Comment:
To be consistent with the function above.
```suggestion
        """Send MQ to indicate finished or failed status for the tracked workflow."""
```